src/shared/ui/Markdown/index.tsx

```diff
@@ -13,7 +13,7 @@ export const Markdown = ({ markdown }: MarkdownProps) => {
   const processedMarkdown = markdownBuilder.extend(markdown)
 
   return (
-    <div>
+    <div id="markdown-wrapper">
```

Comment:
I am not sure about the reason to use id instead of class here

Reply:
I use it for autotest team